Nim/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿Game:
+
+Console.WriteLine("Welcome to Nim!");
+Console.WriteLine("|||||||||||||||||||||||| (24)");
+
+int userInput = 10;
+int matchesNum = 24;
+bool isPlayer = false;
+Random seed = new Random();
+
+while (matchesNum > 0)
+{
+    int Ai = seed.Next(1, 4);
```

Comment:
Technically, you only need to calculate the AiDraw further down in line 30. You can safe some performance by not calculating an ai move if it's not the Ais turn :)